django2pydantic/handlers/relational.py

```
@@ -1,33 +1,43 @@
+    bound=RelatedField[models.Model, models.Model],
+    covariant=True,
+)
+
+
+class RelatedFieldHandler(
+    Generic[TDjangoRelatedField_co], DjangoFieldHandler[TDjangoRelatedField_co], ABC
 ):
     """Base handler for Related fields."""
 
     @override
     def get_pydantic_type_raw(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_pydantic_type_raw`

(ANN201)